distribution/docs/src/main/resources/_contents/_catalog-frameworks/catalog-api-contents.adoc

```diff
@@ -0,0 +1,245 @@
+
+==== Catalog API
+
+The Catalog API is an OSGi bundle (`catalog-core-api`) that contains the Java interfaces for the Catalog components and implementation classes for the Catalog Framework, Operations, and Data components.
+
+===== Catalog API Search Interfaces
+
+The Catalog API includes two different search interfaces.
+
+${ddf-ui} Application Search Interface:: The ${branding} Search UI application provides a graphic interface to return results and locate them on an interactive globe or map.
+
+SSH Search Interface:: Additionally, it is possible to use a client script to remotely access ${branding} via SSH and send console commands to search and ingest data.
+
+===== Catalog Search Result Objects
+
+Data is returned from searches as Catalog Search `Result` objects.
+This is a subtype of Catalog `Entry` that also contains additional data based on what type of sort policy was applied to the search.
+Because it is a subtype of Catalog `Entry`, a Catalog Search `Result` has all Catalog ``Entry``’s fields such as metadata, effective time, and modified time.
```

Comment:
They are needed because of the apostrophe. See [Constrained Quotes](http://asciidoctor.org/docs/user-manual/#when-should-i-use-unconstrained-quotes)